build-analysis/base/openshift-templates/build-analysis.yaml

```diff
@@ -0,0 +1,273 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: build-analysis
+  annotations:
+    description: "Thoth: Build Analysis"
+    openshift.io/display-name: "Thoth: Build Analysis"
+    tags: thoth,ai-stacks,packages,dependencies,package-extract,build
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: >
+      This template defines resources needed to run build analysis on OpenShift.
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+  labels:
+    app: thoth
+    template: build-analysis
+    component: build-analysis
+
+parameters:
+  - name: THOTH_BUILD_ANALYSIS_BASE_IMAGE
+    required: false
```

Comment:
please remove the default ones like `required: false` from all the parameters.
as it causes a sync issue with argocd.

Reply:
Thanks, adjusted. :+1: